paddle/phi/api/yaml/sparse_ops.yaml

```diff
@@ -489,3 +489,15 @@
     func : sync_batch_norm_coo{sparse_coo, dense, dense, dense, dense -> sparse_coo, dense, dense, dense, dense, dense}
     data_type : x
   backward : sync_batch_norm_grad
+
+- op : reshape
+  args : (Tensor x,  IntArray shape)
+  output : Tensor(out)
+  infer_meta :
+    func : ReshapeInferMeta
```

Comment:
## "func : ReshapeInferMeta" , I have used the infermeta function for dense tensor, so there is no need to write new ones.